ui/src/features/components/Report/compareReports.js

```diff
@@ -72,6 +72,44 @@ class CompareReports extends React.Component {
         this.setMergedReports(reportsList);
     };
 
+    onEditSymbol = (value, index) => {
+        const {reportsList} = this.state;
+        const {aggregateReports} = this.props;
+        const newAggregateReports = aggregateReports;
```

Comment:
notice here that it is not really a new object, it is another reference to aggregateReports